src/prime_server.cpp

```diff
@@ -392,6 +395,7 @@ int proxy_t::expire() {
   // just get the time and iterate from the beginning popping off stale ones
   return static_cast<bool>(fifo.size()) + 1;
 }
+
```

Comment:
please revert whitespace change (and the other one below)

Reply:
Done